blog/posts/models.py

```diff
@@ -0,0 +1,14 @@
+from django.db import models
+from users.models import UserModel
+
+class PostModel(models.Model):
+    author = models.ForeignKey(UserModel, on_delete=models.CASCADE)
```

Comment:
Why are you using CASCADE ?
If the user is deleted, do you want to also delete his posts ?